supporting_scripts/analysis-of-endpoint-connections/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id 'java'
+    id 'com.github.johnrengelman.shadow' version '7.1.0'
+}
+
+group 'de.tum.in.www1.artemis'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'com.thoughtworks.qdox:qdox:2.0-M9'
+    implementation 'org.springframework.boot:spring-boot-starter-web:2.5.4'
```

Comment:
Is it maybe possible to use the same version here as in the Artemis gradle properties? Why do we even need a spring dependency here? It does not seem used.

Reply:
We need it to be able to match longer prefixes of the annotations, making parsing the annotations more accurate.